tests/gold_tests/h2/http2_flow_control.replay.yaml

```diff
@@ -88,9 +88,9 @@ sessions:
       headers:
         fields:
         - [ X-Response, first-response ]
-        - [ Content-Length, 28 ]
+        - [ Content-Length, 1200 ]
```

Comment:
I wonder why the size was increased. To make this test stable? Should we do this on 9.2 as well?

Reply:
Good question.

This test tests that ATS correctly handles session and stream windows correctly by passing large bodies with a variety of window sizes. Before HTTP/2 to origin, this test only was significant on the client side. Now we want to exercise these issues on the server side. Thus the body sizes of responses were increased. This is not helpful in 9.2 because we don't support HTTP/2 on the origin side there.

Reply:
I see. Makes sense.